.travis.yml

```diff
@@ -47,11 +47,13 @@ dist: trusty
 sudo: required
 addons:
    postgresql: "9.4"
-   apt:
-    packages:
-    - mysql-server-5.6
-    - mysql-client-core-5.6
-    - mysql-client-5.6
+   mysql: "5.7"
```

Comment:
Perhaps just specifying mysql 5.7 would be enough? (no need for the custom install script above) (@slint)